documentation/frontend/reference/testing_guidelines.md

```diff
@@ -357,15 +357,34 @@ output of failed tests by finding the `trace.zip` for the relevant test in the
 [Playwright Trace Viewer](https://playwright.dev/docs/trace-viewer) to inspect
 these (or open the zip yourself and poke around the files on your own).
 
-For Playwright tests failing in CI, a GitHub comment will appear with a link to
-download an artifact of the `test-results` folder.
+Playwright tests in CI are run with `-u` option by default, this means that
+snapshots will automatically be updated for modified parts of the UI if
+Playwright detects that. See
+[Updating snapshots](/frontend/guides/test.md#updating-snapshots) for more
+reading about this.
+
+When this happens, a GitHub comment will appear with a link to download
+artifacts named in the form `*_snapshot_diff`. The resulting artifacts would
+each contain a gzipped patch of the snapshot changes named
+`snapshot_diff.patch.gz`, once downloaded, they can be decompressed and applied
+to your working branch by running:
+
+`cat snapshot_diff.patch.gz | ov gunzip -kc - | git apply`
```

Comment:
I [downloaded](https://github.com/WordPress/openverse/actions/runs/9798106098/job/27055999711?pr=4600) it as `playwright_vr_snapshot_diff.zip` so I had to decompress it twice (first from a .zip and then from a .gz). It seems that the GitHub actions always do the compression for downloads, so we can skip that part.

```suggestion
`cat playwright_vr_snapshot_diff.zip | ov unzip -kc - | git apply`
```

Reply:
Nice!

Reply:
Thanks for pointing this out @krysal. A slight change to your suggestion would be:
 `cat *_snapshot_diff.zip | ov unzip -kc - | git apply`
 This is because the prefix `*` represents the current playwirght test. It is dynamic and auto generated in the workflow using a matrix. I'd go ahead and note this on the doc too.